api-extractor/src/definitions/ApiProperty.ts

```diff
@@ -15,8 +15,15 @@ class ApiProperty extends ApiMember {
     this.kind = ApiItemKind.Property;
 
     const declaration: any = options.declaration as any; /* tslint:disable-line:no-any */
-    if (declaration && declaration.type) {
-      this.type = declaration.type.getText();
+    if (declaration) {
+      if (declaration.type) {
+        this.type = declaration.type.getText();
+      } else {
+        this.reportError('Property type not declared');
```

Comment:
Can you add a unit test for this case?  I want to see what it looks like in the *.api.ts file